src/tests/system/tests/test_feature_presence.py

```diff
@@ -0,0 +1,86 @@
+"""
+SSSD Feature presence suite
```

Comment:
SSSD Feature Presence Tests.

Because it's a title, any word longer than 3 characters should be capitalized. This is mostly consistent in the test files.

Reply:
I copied this from Jakub's PR. In any case, I have updated it.